mono/mini/debugger-agent.c

```diff
@@ -9509,17 +9519,40 @@ string_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
 
 	return ERR_NONE;
 }
+static void create_file_to_check_memory_address (void) 
+{
+	if (file_check_valid_memory > 0)
+		return;
+	char *file_name = g_strdup_printf ("debugger_check_valid_memory.%d", getpid());
+	filename_check_valid_memory = g_build_filename (g_get_tmp_dir (), file_name, (const char*)NULL);
+	file_check_valid_memory = open(filename_check_valid_memory, O_CREAT | O_WRONLY | O_APPEND, S_IWUSR);
```

Comment:
filename_check_valid_memory  is leaked.

Reply:
it's being deallocated on mono_debugger_agent_cleanup